### PR TITLE
complete todo

### PR DESCRIPTION
**Pull request by kepelivi**
Users are able to set a todo's 'done' status to completed. Functionality and validation are covered with tests.

### CODE REVIEW

File: src/validate.js [hunk 1]
```diff
@@ -21,4 +21,21 @@ export function validateFindByStatusParam(params) {
...
     throw new AppError('Invalid status. Use "done" or "not-done".');
   }
+}
+
+export function validateCompleteParams(params) {
+  if (!params || params.length === 0) {
+    throw new AppError("Id is required.");
+  }
+
+  if (typeof params !== 'string' && typeof params !== 'number') {
```
Comment (GMSzebeni) [inline src/validate.js]: this checking is not necessary. i looked into this and command line arguments are always comming in as type strings. so there is also no need to validate boolean comming in in validate.spec.js.

File: src/validate.js
Comment (GMSzebeni) [inline src/validate.js:1]: if i am typing the command: node index.js complete 2 - it gets stuck on line 31 and I get the error: Id must be a numeric type. so the command does not work for me. it works if lines 31-33 are deleted.

File: src/validate.spec.js [hunk 2]
```diff
@@ -82,4 +82,37 @@ describe('validateFindByStatusParam', () => {
...
+  it('should throw error when param is null', () => {
+    const param = null;
+
+    expect(() => validateCompleteParams(param))
+      .toThrow('Id is required.');
+  });
+
+  it('should throw error when param is not a number', () => {
+    const param1 = "2two";
+    const param2 = true;
```
Comment (GMSzebeni) [inline src/validate.spec.js]: not necessary to check - see my comment in validate.js

**Review by GMSzebeni — CHANGES_REQUESTED** [review]

**Review by GMSzebeni — APPROVED** [review]

**Review by BlazsejKrisz — APPROVED** [review]
Seems good to me. Great job!

**Pull request merged**